### ISSUE TITLE
Is it working with rails 3?

### ISSUE THREAD

**Issue by mcmegavolt** [richpeck/exception_handler#18]
After bundle update cannot get work my rails 3 application now ((

```ruby
undefined method `setup' for ExceptionHandler:Module (NoMethodError)
```

**Comment by richpeck** [comment]
Let's have a look - we had a small issue with this previously, I think I may know the issue

**Comment by mcmegavolt** [comment]
Thanks! Waiting for the solution, it's critical for me (((

**Comment by richpeck** [comment]
Almost got a new version for you - many changes

**Comment by mcmegavolt** [comment]
Wow! Great! ))

**Comment by richpeck** [comment]
Okay just published version `0.4.6` - I am working on the Readme updates for you now.

**Comment by mcmegavolt** [comment]
Awesome! Can't wait to check )))
